lib/travis/build/script/r.rb

```diff
@@ -126,7 +126,7 @@ def configure
 
                 # R-devel builds available at mac.r-project.org
                 if r_version == 'devel'
-                  r_url = "https://mac.r-project.org/el-capitan/R-devel/R-devel-el-capitan.pkg"
+                  r_url = "http://mac.r-project.org/high-sierra/R-4.0-branch/R-4.0-branch.pkg"
```

Comment:
Did you mean to make this http rather than https?

Reply:
Whoops, good catch. Sorry.

Reply:
```suggestion
                  r_url = "https://mac.r-project.org/high-sierra/R-4.0-branch/R-4.0-branch.pkg"
```